src/Library.php

```diff
@@ -143,15 +143,9 @@ public static function register_blocks() {
 	 * Register payment methods.
 	 */
 	public static function register_payment_methods() {
-		$integrations = [
-			'Stripe',
-			'Cheque',
-		];
-		foreach ( $integrations as $class ) {
-			$class    = __NAMESPACE__ . '\\Payments\\Integrations\\' . $class;
-			$instance = new $class();
-			$instance->register_payment_method();
-		}
+		$payment_method_registry = Package::container()->get( PaymentMethodRegistry::class );
+		do_action( 'wc_blocks_library_payment_method_type_registration', $payment_method_registry );
+		$payment_method_registry->initialize();
```

Comment:
This is the mechanism by which payment method extensions will register their types.  They hook into the action and then receive an reference to the `$payment_method_registry`  from which they can register their instantiated classes (extending `AbstractPaymentMethodType` or implementing `PaymentMethodTypeInterface`).